<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите координаты центра фигуры:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="xSpinBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>67</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ySpinBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>67</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="zSpinBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>67</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите размеры параллелепипеда:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>67</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>67</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_6">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>150</y>
      <width>67</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>c</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addCubeBtn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>261</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить параллелепипед</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите радиус сферы:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="radiusSpinBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>67</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addSphereBtn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>261</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить сферу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>591</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать сложную фигуру</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>300</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>340</y>
      <width>261</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить фигуру в начале</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>261</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить фигуру в конце</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
